smarts/core/bubble_manager.py

```diff
@@ -708,7 +708,11 @@ def _prepare_sensors_for_agent_control(
         else:
             goal = EndlessGoal()
         mission = Mission(start=Start(vehicle.position[:2], vehicle.heading), goal=goal)
-        plan.create_route(mission)
+        try:
+            plan.create_route(mission)
+        except PlanningError:
+            plan.mission = None
```

Comment:
I have done test and found that bubble works fine after commenting out this line of code, and the speed is quite acceptable. It probably because we defines that `plan.mission = None` in initialization and we don't need to assign value again? 

**Error Message**
> Traceback (most recent call last):
  File "run_scripts/visualize_smarts.py", line 196, in <module>
    raise e
  File "run_scripts/visualize_smarts.py", line 192, in <module>
    experiment(exp_specs)
  File "run_scripts/visualize_smarts.py", line 62, in experiment
    visualize(variant, env)
  File "run_scripts/visualize_smarts.py", line 129, in visualize
    next_observation_n, reward_n, terminal_n, env_info_n = env.step(action_n)
  File "/home/z50024513/Desktop/SMARTS/SMARTS_Benchmark/ILSwiss/rlkit/envs/wrappers.py", line 155, in step
    observation_n, reward_n, done_n, info_n = self._wrapped_env.step(action_n)
  File "/home/z50024513/Desktop/SMARTS/SMARTS_Benchmark/ILSwiss/rlkit/envs/wrappers.py", line 105, in step
    return self._wrapped_env.step(scaled_action_n)
  File "/home/z50024513/Desktop/SMARTS/SMARTS_Benchmark/ILSwiss/rlkit/env_creators/smarts/smarts_env.py", line 44, in step
    return self._env.step(action_n)
  File "/home/z50024513/Desktop/SMARTS/SMARTS_Benchmark/smarts-imitation/smarts_imitation/envs/bubble_core.py", line 371, in step
    raw_observation_n, reward_n, self.done_n, _ = self.smarts.step(scaled_action_n)
  File "/home/z50024513/Desktop/SMARTS/SMARTS_Benchmark/SMARTS/smarts/core/smarts.py", line 220, in step
    return self._step(agent_actions, time_delta_since_last_step)
  File "/home/z50024513/Desktop/SMARTS/SMARTS_Benchmark/SMARTS/smarts/core/smarts.py", line 285, in _step
    self._bubble_manager.step(self)
  File "/home/z50024513/Desktop/SMARTS/SMARTS_Benchmark/SMARTS/smarts/core/bubble_manager.py", line 487, in step
    self._handle_transitions(sim, self._cursors)
  File "/home/z50024513/Desktop/SMARTS/SMARTS_Benchmark/SMARTS/smarts/core/bubble_manager.py", line 554, in _handle_transitions
    sim, cursor.vehicle_id, cursor.bubble.actor, cursor.bubble
  File "/home/z50024513/Desktop/SMARTS/SMARTS_Benchmark/SMARTS/smarts/core/bubble_manager.py", line 628, in _airlock_social_vehicle_with_social_agent
    sim, vehicle_id, agent_id, interface, bubble
  File "/home/z50024513/Desktop/SMARTS/SMARTS_Benchmark/SMARTS/smarts/core/bubble_manager.py", line 739, in _prepare_sensors_for_agent_control
    plan.mission = None
AttributeError: can't set attribute

Reply:
> And it works well if we just remove the line `plan.mission = None` in our experiment.

Very sorry, I think I added that line just after testing even though it is an immutable property, I have addressed it properly.